packages/amplify_lints/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: amplify_lints
+description: Lints used in developing Flutter packages and plugins.
+version: 1.0.0
```

Comment:
Would it make sense to add `publish_to: "none"` in this file?

Reply:
Hmm I had actually foreseen publishing it, my thought being that we may have projects which live outside this repo eventually